ntbs-service/Migrations/20211104164435_MoveMDRFieldsToMDRModel.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace ntbs_service.Migrations
+{
+    public partial class MoveMDRFieldsToMDRModel : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<string>(
+                name: "ExpectedTreatmentDurationInMonths",
+                table: "MDRDetails",
+                type: "nvarchar(10)",
+                maxLength: 10,
+                nullable: true);
+
+            migrationBuilder.AddColumn<DateTime>(
+                name: "MDRTreatmentStartDate",
+                table: "MDRDetails",
+                type: "datetime2",
+                nullable: true);
+
```

Comment:
I think I need to write a bit of SQL here to migrate over the values that already exist.